[2025-06-25T02:24:28.225+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-25T02:24:28.407+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: clinlytix360_pipeline.log_dashboard_refresh manual__2025-06-25T02:22:47.676421+00:00 [queued]>
[2025-06-25T02:24:28.467+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: clinlytix360_pipeline.log_dashboard_refresh manual__2025-06-25T02:22:47.676421+00:00 [queued]>
[2025-06-25T02:24:28.469+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-06-25T02:24:28.552+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): log_dashboard_refresh> on 2025-06-25 02:22:47.676421+00:00
[2025-06-25T02:24:28.611+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=221) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-25T02:24:28.617+0000] {standard_task_runner.py:63} INFO - Started process 223 to run task
[2025-06-25T02:24:28.619+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'clinlytix360_pipeline', 'log_dashboard_refresh', 'manual__2025-06-25T02:22:47.676421+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/clinlytix_dag.py', '--cfg-path', '/tmp/tmpekf3mtja']
[2025-06-25T02:24:28.634+0000] {standard_task_runner.py:91} INFO - Job 24: Subtask log_dashboard_refresh
[2025-06-25T02:24:28.949+0000] {task_command.py:426} INFO - Running <TaskInstance: clinlytix360_pipeline.log_dashboard_refresh manual__2025-06-25T02:22:47.676421+00:00 [running]> on host 8c1f13682a11
[2025-06-25T02:24:29.383+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ajayc' AIRFLOW_CTX_DAG_ID='clinlytix360_pipeline' AIRFLOW_CTX_TASK_ID='log_dashboard_refresh' AIRFLOW_CTX_EXECUTION_DATE='2025-06-25T02:22:47.676421+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-25T02:22:47.676421+00:00'
[2025-06-25T02:24:29.393+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-25T02:24:29.402+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-25T02:24:29.415+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Dashboard ready to serve latest data."']
[2025-06-25T02:24:29.473+0000] {subprocess.py:86} INFO - Output:
[2025-06-25T02:24:29.503+0000] {subprocess.py:93} INFO - Dashboard ready to serve latest data.
[2025-06-25T02:24:29.504+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-06-25T02:24:29.516+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-25T02:24:29.634+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=clinlytix360_pipeline, task_id=log_dashboard_refresh, execution_date=20250625T022247, start_date=20250625T022428, end_date=20250625T022429
[2025-06-25T02:24:29.792+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-25T02:24:29.868+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-25T02:24:29.876+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
